wgengine/magicsock/magicsock.go

```diff
@@ -80,6 +82,12 @@ const (
 	socketBufferSize = 7 << 20
 )
 
+// CaptureFunc describes a callback to record packets when
+// debugging packet-capture. Such callbacks must not take
+// ownership of the provided data slice: it may only copy
+// out of it within the lifetime of the function.
+type CaptureFunc func(capture.Path, time.Time, []byte)
```

Comment:
why's this here in magicsock? if you're already importing the capture package?

Reply:
Just keeping the type where its used. We already have equivalents in tstun and wgengine, if we want to centralize it maybe we should rip the same out of there too?

Reply:
yeah, one place would be ideal